kratos/utilities/binbased_fast_point_locator.h

```diff
@@ -346,6 +346,28 @@ class BinBasedFastPointLocator
     ///@name Private Operations
     ///@{
 
+    /**
+     * @brief This operation is defined to the the corresponding container type
+     * @param rModelPart The model part to get the element container
+     * @return The corresponding element array
+     */
+    static inline PointerVectorSet<Element, IndexedObject>::ContainerType GetContainerType(ModelPart& rModelPart, PointerVectorSet<Element, IndexedObject>::ContainerType rContainerArray)
+    {
+        rContainerArray = rModelPart.ElementsArray();
+        return rContainerArray;
```

Comment:
Why passing in rContainerArray and returning it ?

Reply:
I can just pass it, but think that the double structure passing returning is common in Kratos (many get values)